src/app/services/local-storage.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LocalStorageService {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  setLSItem(key: string, value: string | { [key: string]: any }): void {
+    const storageValue = JSON.stringify(value);
```

Comment:
It's better to check value type here, to avoid extra JSON.stringify calls and possibal errors:
`typeof value === 'string' ? value :  JSON.stringify(value)`